src/HypothesisClient/Credentials/JWTSigningCredentials.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace eLife\HypothesisClient\Credentials;
+
+use Firebase\JWT\JWT;
+
+class JWTSigningCredentials extends Credentials
+{
+    private $expire;
+
+    public function __construct(string $clientId, string $clientSecret, string $authority, int $expire = 600)
+    {
+        parent::__construct($clientId, $clientSecret, $authority);
+        $this->expire = $expire;
+    }
+
+    public function getExpire() : int
+    {
+        return $this->expire;
+    }
+
+    public function getJWT(string $username) : string
+    {
+        $now = $_SERVER['REQUEST_TIME'];
```

Comment:
`$_SERVER` can't be used from the cli (hence from tests) without overriding global variables; inject a `integer` timestamp in the constructor

Reply:
(as this is not used in the long-running process)